docs/users.md

```diff
@@ -522,6 +530,63 @@ vcluster-0                                          2/2     Running   0
 
 The nginx pod is the one with the name `nginx-x-default-x-vcluster`.
 
+## Working with an external control plane
+
+In this section, we will show an example of creating an external control plane to integrate
+a kind cluster named `ext1` into the same Docker network as your Kubeflex hosting cluster. This setup 
+requires that both clusters are accessible from each other within the shared Docker network.
+
+### Prerequisites
+Ensure that both your hosting and adopted clusters are on the same Docker network. For 
+clusters using the default `kind` docker network, execute the following command to 
+verify connectivity:
+
+```shell
+docker network inspect kind | jq '.[].Containers | to_entries[] | .value.Name'
+```
+
+The output will list the container names for your clusters. For example:
+
+```shell
+"ext1-control-plane" represents the external cluster you wish to adopt.
+"kubeflex-control-plane" represents the Kubeflex hosting cluster.
+```
+
+Once verified, the endpoint for the adopted cluster should be set as follows:
+
+```shell
+https://ext1-control-plane:6443
+```
+
+If you're not utilizing the default `kind` network, you'll need to identify which Docker networks are available. List all networks with this command:
+
+```shell
+docker network ls
+```
+
+Inspect any specific network to confirm its configuration using docker network inspect <network_name>.
+
+### Adopting the external cluster
+
+To set up the external cluster ext1 as a control plane named cpe, use the following command:
+
+```shell
+$ kflex adopt -c kind-ext1 -u https://ext1-control-plane:6443
```

Comment:
The meanings of "c" in `-c` and "u" in `-u` are not really obvious, I would use the long flag names here.